util/util.go

```diff
@@ -1014,3 +1014,19 @@ func VerifySnapshotLabels(labels map[string]string) error {
 	}
 	return nil
 }
+
+func IsTagsEqual(tags1, tags2 []string) bool {
```

Comment:
Please add a doc comment. This is only really testing the tags are equal, but determine if tags2 is subset of tags1 instead.